migration-engine/migration-engine-tests/tests/errors/error_tests.rs

```diff
@@ -515,53 +516,64 @@ async fn native_types_are_not_allowed_in_migration_engine() {
 }
 
 #[tokio::test]
-async fn one_can_allow_all_preview_features() {
```

Comment:
Is this test completely gone? (wouldn't bother me too much, we'll have native types soon)

Reply:
We still test `nativeTypes` gatekeeping separately, but this tested both: `microsoftSqlServer` and `nativeTypes`.